### PR TITLE
Nomic finalization

### PR DESCRIPTION
**Pull request by cwlittle**
## What is the purpose of the change
Finalize changes for Nomic integration

### CODE REVIEW

File: packages/web/integrations/nomic/transfer.tsx [hunk 1]
```diff
@@ -387,19 +387,13 @@ const NomicTransfer: FunctionComponent<
...
-                    <span>{t("assets.nomic.bridgeFee")}</span>
-                    <SkeletonLoader
-                      isLoaded={Boolean(bridgeInfo?.bridgeFeeRate)}
-                    >
-                      <span>{`${(
-                        (bridgeInfo?.bridgeFeeRate as number) * 100
-                      ).toFixed(2)}%`}</span>
+                    <span>{t("assets.nomic.bitcoinMinerFee")}</span>
+                    <SkeletonLoader isLoaded={Boolean(bridgeInfo)}>
+                      <span>{`~0.00002 BTC`}</span>
```
Comment (JoseRFelix) [inline packages/web/integrations/nomic/transfer.tsx:396]: Is this expected?

Comment (cwlittle) [reply packages/web/integrations/nomic/transfer.tsx:396]: Yes. The bridge fee only applies to deposits.

Comment (JoseRFelix) [reply packages/web/integrations/nomic/transfer.tsx:396]: Could this change in the future? If so, how can we keep this updated?

Comment (cwlittle) [reply packages/web/integrations/nomic/transfer.tsx:396]: We have some relayer updates that are coming soon that will provide this value. We'll open a PR to get that integrated.

**Review by JoseRFelix — APPROVED** [review]

**Pull request merged**